GameOfLife/Calculations/GameFieldSize.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GameOfLife
+{
+    class GameFieldSize
```

Comment:
Add appropriate Public/Private access modifier

Reply:
Set to public access modifier.